connectors/sources/sharepoint_server.py

```diff
@@ -844,3 +840,13 @@ async def download_func(self, response_data):
         it instead contains a key with bytes in its response.
         """
         yield bytes(response_data, "utf-8")
+
+    async def chunked_download_func(self, download_func):
```

Comment:
Why have a different function implementation here?

Reply:
Because the one in source.py assumes it's receiving a aiohttp response object (response.content.iter_chunked), which is not the case when using httpx.

Reply:
Ah makes sense, thanks for the explanation!